kuma/static/styles/minimalist/main.scss

```diff
@@ -23,6 +23,7 @@ Main stylesheet for the site, standalone
 @import '../components/_tags.scss';
 @import '../components/_errorlist.scss';
 @import '../components/_newsletter.scss';
+@import './components/_newsletter.scss';
```

Comment:
Hehe. Perhaps it's time to name those two things differently :)

Reply:
Curious why the minimalist rules in `_newsletter.scss` aren't merged with the other `_newsletter.scss`?

Reply:
Just have not had the time :(

Reply:
It's okay :)